public/pages/Emails/components/tables/SendersTable.tsx

```diff
@@ -120,6 +122,9 @@ export class SendersTable extends Component<
     if (!_.isEqual(prevQuery, currQuery)) {
       await this.refresh();
     }
+    if (isDataSourceChanged(this.props, prevProps)) {
```

Comment:
Nitpick:
This check could be worked into line check on line 125 by changing the check to
```
if (!_.isEqual(prevQuery, currQuery) || isDataSourceChanged(this.props, prevProps)) { ...